examples/README.md

```diff
@@ -4,7 +4,7 @@ To build and run these examples:
 1. `cd $example-dir`
 1. `npm install ../.. # this installs the libhoney you built above`
 1. `npm install`
-1. `HONEY_WRITE_KEY=YOUR_WRITE_KEY npm start`
+1. `HONEYCOMB_API_KEY=YOUR_WRITE_KEY npm start`
```

Comment:
Since this PR is for the major version bump in dropping support for old Node, I think we should move these example changes to a different PR to keep the focus on the major change.

Reply:
Cool, reverted and I'll make a new PR with those.

Reply:
@JamieDanielson https://github.com/honeycombio/libhoney-js/pull/309 ✨